lib/src/screens/initial_setup/initial_setup_screen.dart

```diff
@@ -36,7 +36,7 @@ import 'package:solidpod/src/screens/initial_setup/initial_setup_screen_body.dar
 
 /// Numeric variables used in initial setup page.
 
-const double normalLoadingScreenHeight = 200.0;
+// const double normalLoadingScreenHeight = 200.0;
```

Comment:
How does this affect the doc string just above? It may look somewhat odd in the generated docs?